metadata-ingestion/examples/demo_data/dbt_catalog.json

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
Not sure if I should leave these (dbt_catalog.json / dbt_manifest.json) in `demo_data` or delete them